turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -641,12 +634,6 @@ pub(crate) async fn analyse_ecmascript_module_internal(
         import_references.push(r);
     }
 
-    for r in import_references.iter_mut() {
-        // Resolving these references here avoids many resolve wrapper tasks when
-        // passing that to other turbo tasks functions later.
-        *r = r.resolve().await?;
-    }
-
```

Comment:
This is a performance hack (not needed for correctness) that appears to still be relevant here, as `import_references` is still a `Vec<Vc<EsmAssetReference>>`